api/src/main/java/org/pmiops/workbench/access/AccessModuleServiceImpl.java

```diff
@@ -128,11 +114,38 @@ private static DbAccessModule getDbAccessModuleFromApi(
                     "There is no access module named: " + apiAccessModule.toString()));
   }
 
-  private static AccessModuleName clientAccessModuleToStorage(AccessModule s) {
-    return CLIENT_TO_STORAGE_ACCESS_MODULE.get(s);
+  /**
+   * Calculates the module expiration time.
+   *
+   * <p>The value is only present when:
+   *
+   * <ul>
+   *   <li>The module is expirable.
+   *   <li>The module was completed(CompletionTime is not null).
+   *   <li>The module is not bypassed(BypassTime is null).
+   *   <li>Access annual renewal is enabled(enableAccessRenewal is true).
+   * </ul>
+   */
+  private Optional<Timestamp> getExpirationTime(DbUserAccessModule dbUserAccessModule) {
+    if (!dbUserAccessModule.getAccessModule().getExpirable()
```

Comment:
I like this separation, but I think we actually want `||` for all of these.  I'd also prefer to have the `enableAccessRenewal` check first.

Reply:
Please add an AccessModuleServiceTest for the case where enableAccessRenewal=false

Reply:
Good catch! Also added a test for that.